src/graph/tree/collapsable-tree-node/collapsable-tree-node.tsx

```diff
@@ -0,0 +1,32 @@
+import React, { ReactNode, HTMLAttributes } from 'react';
+import classNames from 'classnames';
+import AnimateHeight from 'react-animate-height';
+import { TreeLayer, TreeNodeProps } from '@teambit/base-ui.graph.tree.recursive-tree';
+import { indentClass, indentStyle } from '@teambit/base-ui.graph.tree.indent';
+import styles from './collapsable-tree-node.module.scss';
+
+export type CollapsableTreeNodeProps = {
+  /**
+   * The title to be rendered and to be clicked to open the content.
+   */
+  title: ReactNode;
+
+  /**
+   * If the content is open or not.
+   */
+  isOpen?: boolean;
+} & Omit<HTMLAttributes<HTMLDivElement>, 'title'> &
+  TreeNodeProps;
+
+export function CollapsableTreeNode({ title, isOpen = false, node, depth, className }: CollapsableTreeNodeProps) {
+  return (
+    <div className={className}>
+      {title}
+      <AnimateHeight height={isOpen ? 'auto' : 0}>
```

Comment:
maybe not render this whole section when there are no children? (or when children.length == 0)

Reply:
done